### PR TITLE
megan point 2 delivery_employee

### PR DESCRIPTION
**Pull request by 40175096**
created delivery_employee table to satisfy point 2

### CODE REVIEW

File: 002CreateDeliveryEmployee.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+USE ThePrimaryKeys_DylanC;
+
+CREATE TABLE delivery_employee (
+	delivery_employee_id smallint unsigned NOT NULL, 
+    employee_id smallint unsigned NOT NULL
+);
+
+ALTER TABLE delivery_employee
+	ADD PRIMARY KEY(delivery_employee_id);
```
Comment (shaunganley) [inline 002CreateDeliveryEmployee.sql]: why do this separately, can you do it in the create table command above?

File: 002CreateDeliveryEmployee.sql [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    
+ALTER TABLE delivery_employee
+ADD CONSTRAINT fk_delivery_employee_employee_id
+FOREIGN KEY (employee_id)
+REFERENCES employee(employee_id);
```
Comment (shaunganley) [inline 002CreateDeliveryEmployee.sql]: can we do this in 1 command?

**Review by shaunganley — CHANGES_REQUESTED** [review]
comments added. can you also include some insert statements and a select statement to prove the feature works

File: 002CreateDeliveryEmployee.sql [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+USE ThePrimaryKeys_DylanC;
+
+-- creating delivery_employee table
+CREATE TABLE delivery_employee (
+    employee_id smallint unsigned AUTO_INCREMENT PRIMARY KEY,
+    FOREIGN KEY (employee_id)
+	REFERENCES employee(employee_id)
+);
+
+DESCRIBE delivery_employee;
```
Comment (glenn-hamilton) [inline 002CreateDeliveryEmployee.sql]: Is the describe needed here?

File: 002CreateDeliveryEmployee.sql [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+-- add employees to employee table then add them to delivery_employee table to make them delivery employees
+INSERT INTO employee
```
Comment (glenn-hamilton) [inline 002CreateDeliveryEmployee.sql]: If you run this in isolation will this not fail until the scrips that have the employee have been created?

File: 002CreateDeliveryEmployee.sql [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+INSERT INTO delivery_employee
+(employee_id) 
+VALUES (4);
+
+INSERT INTO delivery_employee
+(employee_id) 
+VALUES (5);
+
+SELECT * FROM delivery_employee;
```
Comment (glenn-hamilton) [inline 002CreateDeliveryEmployee.sql]: Can you be more specific with the `SELECT`

File: 002CreateDeliveryEmployee.sql [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+VALUES (4);
+
+INSERT INTO delivery_employee
+(employee_id) 
+VALUES (5);
+
+-- create right join with employee table to see id of delivery employee and name
+SELECT delivery_employee.employee_id, CONCAT(first_name, ' ', last_name) AS `Name` 
+FROM delivery_employee
+RIGHT JOIN employee
```
Comment (shaunganley) [inline 002CreateDeliveryEmployee.sql:29]: this should be a left join as you only want to show delivery employees

**Review by shaunganley — APPROVED** [review]

**Pull request merged**